### PR TITLE
fix(client): improve full stack certificate table

### LINKED ISSUE #50212: Full stack certification rendering oddly
...I am pretty sure the Full Stack Cert should not show "Solution":
image

_Originally posted by @ShaunSHamilton in https://github.com/freeCodeCamp/freeCodeCamp/issues/50206#issuecomment-1527511751_

### PR DESCRIPTION
**Pull request by SergenKaraoglan**
- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #50212

Removed solution column and header. Also removed certification header and renamed certification titles.

Before:
235151537-94846fe8-c48b-40a3-b524-b277d512d39b

After:
Screenshot 2023-04-29 at 21 24 30

### CODE REVIEW

**Comment by bbsmooth** [comment]
Technically, this isn't tabular data any more and should be a list instead of a table. But if we are going to keep it coded as a table then I'll repeat what I said in the issue, this table should have a `th` (which I'm guessing would be "Certification"). We can wrap a span around the text and use `sr-only` if we don't want it to show on the page. Screen reader users expect tables to have headings. I might go as far as to call it an accessibility anti-pattern not to have them. And if we add the `th` then we would not need to add "Certification" after each one which would reduce verbosity just a little.

OK, I'll shut up about this now. In the scheme of things it's a minor issue. WCAG does not require a table to have headings. But I think most accessibility professionals would consider it odd not to have them.

**Comment by SergenKaraoglan** [comment]
> Technically, this isn't tabular data any more and should be a list instead of a table. But if we are going to keep it coded as a table then I'll repeat what I said in the issue, this table should have a `th` (which I'm guessing would be "Certification"). We can wrap a span around the text and use `sr-only` if we don't want it to show on the page. Screen reader users expect tables to have headings. I might go as far as to call it an accessibility anti-pattern not to have them. And if we add the `th` then we would not need to add "Certification" after each one which would reduce verbosity just a little.
> 
> OK, I'll shut up about this now. In the scheme of things it's a minor issue. WCAG does not require a table to have headings. But I think most accessibility professionals would consider it odd not to have them.

Have followed the above suggestions. Let me know what you think.

**Review by gikf — APPROVED** [review]
LGTM :+1:

File: client/src/client-only-routes/show-project-links.tsx [hunk 1]
```diff
@@ -164,8 +164,9 @@ const ShowProjectLinks = (props: ShowProjectLinksProps): JSX.Element => {
       <Table striped>
         <thead>
           <tr>
-            <th>{t('profile.challenge')}</th>
-            <th>{t('settings.labels.solution')}</th>
+            <th>
+              <span className='sr-only'>Certification</span>
```
Comment (Sboonny) [inline client/src/client-only-routes/show-project-links.tsx:168]: This isn't using a translatable string, we can use `Certifications` from translation.json and make it translatable.

https://github.com/freeCodeCamp/freeCodeCamp/blob/f5b86a197b2afc05e67e0bcae5207ff62acb4a93/client/i18n/locales/english/translations.json#L156

```suggestion
              <span className='sr-only'>{t('settings.headings.certs')}</span>
```

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @SergenKaraoglan, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Pull request merged**